### ISSUE TITLE
'latest' symbolic link for download

### ISSUE THREAD

**Issue by i2** [build2/build2#290]
I suggest adding a symlink called _latest_ at [download.build2.org](https://download.build2.org/?C=M;O=D), so that downloading the latest version could be better scripted. Right now, one has to specify a version like 0.16.0.

**Comment by boris-kolpackov** [comment]
We could do that but there is already a mechanism to this end that has extra benefits: the `toolchain.sha256` file: https://download.build2.org/toolchain.sha256. It always contain paths (relative to its location) to the latest toolchain files plus their SHA256 sums (the format of that file is the same as what's produced by the `sha256sum` utility on Linux). We use it ourselves to bootstrap our CI bots. Would you be able to use that instead of the symlink?

**Comment by i2** [comment]
Thanks for the response!

This came to my mind as normally projects have a latest label/folder/symlink. I'll go a step forward and suggest adding latest label in [GitHub releases](https://github.com/build2/build2/tags) too.

What you mentioned would be useful for me. In this case, could you please add the instructions to the install page at build2.org so people can readily access it?

**Comment by boris-kolpackov** [comment]
> In this case, could you please add the instructions to the install page at build2.org so people can readily access it?

Ok, I've added a `README` file to the download directory with some details: https://download.build2.org/README

And I've added a note to the download page (the install page references the download page at the bottom): https://build2.org/download.xhtml

> I'll go a step forward and suggest adding latest label in [GitHub releases](https://github.com/build2/build2/tags) too.

Hm, `build2` consists of quite a few repositories and maintaining the `latest` tag (I assume you mean a `git` tag) will be a bit of a chore. Plus, I am not sure modifying a git tag is a good idea, and even if it is, I am not sure it can be done atomically. So this will require more thinking. Maybe/later.

> If that instruction could be universal among platforms/OSes, it'd be golden!

I don't think that's possible if you include Windows. There is just no common shell language among UNIX and Windows (without assuming that the use has MSYS or some such, which we cannot make).
